src/TLC.sol

```diff
@@ -0,0 +1,308 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+
+import {Ownable} from "openzeppelin-contracts/access/Ownable.sol";
+import {IERC20} from "openzeppelin-contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "openzeppelin-contracts/token/ERC20/utils/SafeERC20.sol";
+
+import {Operators} from "./common/access/Operators.sol";
+
+contract TLC is Ownable, Operators {
+
```

Comment:
Also - can't easily migrate user positions if we need a v2. pros/cons of making this upgradeable?

Reply:
https://github.com/TempleDAO/Temple-Loving-Care/issues/5